.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+# Each virtual machine has the same hardware resources available.
+#
+# - 2-core CPU
+# - 7 GB of RAM memory
+# - 14 GB of SSD disk space
+#
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners
+
+name: CI
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
```

Mega nit:

```suggestion
    runs-on: ubuntu-18.04
```

Looks like they also offer 20.04 as a preview. But I figure we should choose the most recent stable LTS